crates/dvdbnd/src/lib.rs

```diff
@@ -140,6 +143,7 @@ impl DvdBnd {
                     data_cipher.decrypt_blocks(blocks);
                 }
 
+                #[cfg(not(target_family = "windows"))]
```

Comment:
```suggestion
                #[cfg(not(target_family = "unix"))]
```

Reply:
Uh I don't think that's correct?
Although its probably better to do `#[cfg(target_family = "unix")]` as it doesn't seem to be support at all outside of unix

Reply:
> Uh I don't think that's correct?

Based on what? https://github.com/RazrFalcon/memmap2-rs/blob/master/src/lib.rs#L47-L48

Reply:
Think I was staring at a cached version of your suggestion

Reply:
febdc3318e068f2d5fd365c745237814c50f783c should do the trick